{"name": "test_sign_up_invalid_phone_num2", "status": "passed", "description": "the test is verify, that user can't sign up with invalid phone number", "start": 1680348735924, "stop": 1680348736673, "uuid": "65ec8936-19a2-438b-a596-6e1537c04fb8", "historyId": "3b1e7642cb70f5d9b866e5fb6b27472f", "testCaseId": "83d801c547d346e613bdec82c5721eea", "fullName": "tests.chromeTests.testSignUpModal.TestSignUpModal#test_sign_up_invalid_phone_num2", "labels": [{"name": "parentSuite", "value": "tests.chromeTests"}, {"name": "suite", "value": "testSignUpModal"}, {"name": "subSuite", "value": "TestSignUpModal"}, {"name": "host", "value": "krot"}, {"name": "thread", "value": "12616-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.chromeTests.testSignUpModal"}]}